integrated_channels/sap_success_factors/client.py

```diff
@@ -0,0 +1,54 @@
+"""
+Client for connecting to SAP SuccessFactors.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import requests
+import slumber
+
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsGlobalConfiguration
+
+
+class SAPSuccessFactorsAPIClient(slumber.API):
+
+    @classmethod
```

Comment:
I don't see any use of `cls` in this method, so this would likely be better with a `@staticmethod` decorator.